app/routes/issuer.py

```diff
@@ -35,28 +39,51 @@
 
 @router.get("", response_model=List[CredentialExchange])
 async def get_credentials(
-    connection_id: Optional[str] = Query(None),
+    connection_id: Optional[UUID] = Query(None),
```

Comment:
Is it correct to use UUID type here? Are all connection-ids of UUID type? And does it just add some extra validation? If so, may as well implement it everywhere! Because we currently mostly use str type for connection_id. So either str everywhere, or UUID everywhere.